woocommerce/woocommerce-ios
# Shipping Labels: basic UI for Carriers and Rates

Part of #4085 

## Description
In this PR I implemented a basic UI for the Carriers and Rates screen, implementing:
- The main view for Carriers and Rates implemented in SwiftUI.
- A simple view model for Carriers and Rates that fetch the carriers and rates from the remote API, and manage the sync status. 
- The `ShippingLabelCarrierRow` displays the information of every single carrier.
- The selected package is now a computed property in `ShippingLabelFormViewModel`.
- `ShimmerModified` applies an animated `shimmer` effect to any SwiftUI view. In our specific case, it will be used with the `redacted` modifier for showing a ghost cell.
- Added the 3 carriers logos in the app (UPS, USPS, FedEx).

## Testing
1. Launch the app in debug mode (because of the feature flag).
2. Open an order detail that is eligible for the shipping label.
3. Press the button "Create Shipping Label".
4. Press the button "Continue" in Create Shipping Label form, under the **Ship From** cell. Confirm the address.
5. Press the button "Continue" in Create Shipping Label form, under the **Ship To** cell. Confirm the address.
6. Press the button "Continue" in Create Shipping Label form, under the **Package Details** cell.
7. You should see the screen that shows the Package Details screen and all the items that will be shipped.
8. Press the "Package selected" row. 
9. Select a package, then press "Done".
10. Add a Total package weight, press "Done".
11. The "Shipping Carrier and Rates" cell should be enabled now. Press it.
12. You should see the new screen "Carrier and Rates". It should display first the ghost cells, then the data fetched from the API (the data showed are still not formatted).

## GIF
<img src="https://user-images.githubusercontent.com/495617/118838185-c51cd000-b8c5-11eb-9869-f583a058dcdc.gif" width=300 />



Update release notes:
